mlflow/utils/env.py

```diff
@@ -8,3 +14,39 @@ def get_env(variable_name):
 def unset_variable(variable_name):
     if variable_name in os.environ:
         del os.environ[variable_name]
+
+
+class EnvironmentVariable:
+    def __init__(self, name, type, default):
+        self.name = name
+        self.type = type
+        self.default = default
+
+    def get(self):
+        val = os.getenv(self.name)
+        if val:
+            try:
+                return self.type(val)
+            except Exception as e:
+                raise ValueError(
+                    f"Parse environment config {self.name}'s value '{val}' failed. (error: {repr(e)})"
```

Comment:
Do we need this try-catch?

Reply:
Let's raise `MlflowException` with `INVALID_PARAMETER_VALUE` as @jinzhang21 suggested in https://github.com/mlflow/mlflow/pull/5745#discussion_r914417045

Reply:
This can make the error message more meaningful.
Otherwise we might get error like `ValueError: invalid literal for int() with base 10: 'ab'`